lib/chai/core/assertions.js

```diff
@@ -92,18 +117,19 @@ module.exports = function (chai, _) {
   /**
    * ### .nested
    *
-   * Sets the `nested` flag, later used by the `property` assertion.
+   * Enables dot- and bracket-notation in all `.property` and `.include`
```

Comment:
Not sure there's enough value with linking `pathval`. The value in linking `deep-eql` and `type-detect` is that they both have separate documentation that list rules and behaviors that are relevant to someone writing an assertion using those flags, but the documentation for `pathval` just provides documentation on a bunch of functions that are used when using the module independently of Chai. Since we're moving in the direction of having a bunch of micro libraries, I think we should only link those that provide direct relevant information to the Chai user.